onnx_tf/handlers/backend/eye_like.py

```diff
@@ -0,0 +1,42 @@
+import copy
+import numpy as np
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_func
+
+
+@onnx_op("EyeLike")
+@tf_func(tf.pad)
```

Comment:
I think we should not use `tf.pad` here. Keep it blank and pad the output in this file should be better.
`@tf_func` seems meet the limitation. Need more discussion.